modules/react/src/components/basic-controls.js

```diff
@@ -28,20 +28,20 @@ import {
 } from '@hubble.gl/core';
 import EncoderDropdown from './encoder-dropdown';
 
-export default function BasicControls({adapter, busy, setBusy, encoderSettings}) {
+export default function BasicControls({adapter, busy, setBusy, encoderSettings, updateCamera}) {
```

Comment:
Can you check to ensure updateCamera is an optional value in both this component and in adapter.getProps?

Want to make sure this feature is "opt in"

Reply:
I'm passing `updateCamera = undefined` at the end of the props in BasicControls. Now if you don't pass an `updateCamera` function in the BasicControls component it will use the default keyframes. 
The only change in adapter.getProps was that `this.enabled` was added at the end of the check:

```
// Animating the camera is optional, but if a keyframe is defined then viewState is controlled by camera keyframe.
    if (this.scene && this.scene.keyframes.camera && this.enabled) {

      props.controller = false;
      props.viewState = this._getViewState();
    }
```
What should I change about this?